packages/beta/src/__tests__/api/alertsApi.int.spec.ts

```diff
@@ -244,55 +245,34 @@ describe('alerts api', () => {
         property: 'createdTime',
         order: 'desc',
       },
-      cursor: '',
     });
     expect(response.items.length).toBe(1);
```

Comment:
No alarm has been created at this point, so why is this expected?
Is it coming from a previous test?